leshan-integration-tests/src/test/java/org/eclipse/leshan/integration/tests/util/RedisSecureIntegrationTestHelper.java

```diff
@@ -34,7 +34,7 @@ protected SecurityStore createSecurityStore() {
         } else {
             jedis = new JedisPool();
         }
-        securityStore = new RedisSecurityStore(jedis);
+        securityStore = new RedisSecurityStore.Builder(jedis).build();
```

Comment:
maybe we can now avoid to do this changes as constructor is back.

Reply:
Yes, of course, you're right.